clients/src/test/java/org/apache/kafka/common/security/ssl/CommonNameLoggingTrustManagerFactoryWrapperTest.java

```diff
@@ -122,7 +122,7 @@ void testNeverExpiringX509Certificate() throws Exception {
                     wrappedCert.hasUnsupportedCriticalExtension());
             // We have just generated a valid test certificate, it should still be valid now
             assertEquals(cert.getBasicConstraints(), wrappedCert.getBasicConstraints());
-            assertEquals(cert.getIssuerDN(), wrappedCert.getIssuerDN());
```

Comment:
why we need those changes?

Reply:
it seens this deprecated method won't cause build error, I will revert it